src/fragments/lib/auth/android/signin_next_steps/10_signin.mdx

```diff
@@ -4,9 +4,78 @@ The `nextStep` property is of enum type `AuthSignInStep`. Depending on its value
 
 <BlockSwitcher>
 
+<Block name="Java">
+
+```java
+try {
+            AWSCognitoAuthSignInOptions options =
```

Comment:
Please fix the indentation of the code inside this try block.

Reply:
done